.github/workflows/release.yaml

```diff
@@ -219,9 +223,18 @@ jobs:
 
       - name: Run Docker and Extract Artifacts
         run: |
-          id=$(docker create build-deb:${{ github.run_id }})
+          id=$(docker create ${{ env.TAG }})
           docker cp $id:/build/${{ github.job }}_${{ needs.validate-preconditions.outputs.version }}.deb ./${{ env.ARTIFACT_NAME }}
 
+      - name: Save image
+        run: docker save -o /tmp/${{ env.ARTIFACT_FILE_NAME }}.tar ${{ env.TAG }}
+  
+      - name: Upload docker artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: docker-${{ matrix.node.env }}
```

Comment:
There is no matrix on this job. Either hardcode to docker-bootstrapper or rename the key to docker-p2p-bootstrapper and use github.job instead.